### PR TITLE
PSMDB-1487 Docs for 8.0

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Comment by radoslawszulgo** [comment]
Change in docs/index.md missing - it still displays 7.0

File: docs/install/tarball.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Install Percona Server for MongoDB from binary tarball
 
-You can find links to the binary tarballs under the *Generic Linux* menu item on the [Percona website](https://www.percona.com/downloads/percona-server-mongodb-7.0/)
+You can find links to the binary tarballs under the *Generic Linux* menu item on the [Percona website](https://www.percona.com/downloads/percona-server-mongodb-6.0/)
```
Comment (sandraromanchenko) [inline docs/install/tarball.md:3]:
```suggestion
You can find links to the binary tarballs under the *Generic Linux* menu item on the [Percona website](https://www.percona.com/downloads/percona-server-mongodb-8.0/)
```

File: mkdocs-base.yml [hunk 2]
```diff
@@ -128,17 +129,22 @@ plugins:
...
 # exclude: # Don't process these files
 #      glob:
 #         - file.md
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
-          output_path: '_pdf/PerconaServerforMongoDB-7.0.pdf'
-          cover_title: 'Server for MongoDB Documentation 7.0'
-          cover_subtitle: 7.0.12-7 (July 23, 2024)
+          output_path: '_pdf/PerconaServerforMongoDB-8.0.pdf'
+          cover_title: 'Server for MongoDB Documentation 8.0'
+          cover_subtitle:
```
Comment (sandraromanchenko) [inline mkdocs-base.yml:147]:
```suggestion
          cover_subtitle: 8.0.1-1 (October 30, 2024)
```

File: docs/psmdb-pro.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Percona Server for MongoDB Pro is a build of Percona Server for MongoDB that contains purpose-built enterprise [features](#features). It is wrapped in packages created and tested by Percona and is available exclusively for Percona customers.
 
-Percona Server for MongpDB Pro is available starting with version [7.0.4-2](release_notes/7.0.4-2.md).
+Percona Server for MongpDB Pro is available starting with version [7.0.4-2](release_notes/https://docs.percona.com/percona-server-for-mongodb/release_notes/7.0.4-2.html).
```
Comment (sandraromanchenko) [inline docs/psmdb-pro.md]:
```suggestion
Percona Server for MongoDB Pro is available starting with version [8.0.1-1](release_notes/https://docs.percona.com/percona-server-for-mongodb/release_notes/8.0.1-1.html).
```

File: docs/psmdb-pro.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Find the list of solutions available in Percona Server for MongoDB Pro builds:
 
 | Name                                | Version added | Description  | 
 | ----------------------------------- | ------------- | -------------
-| [FIPS support ](fips.md)| [7.0.4-2](release_notes/7.0.4-2.md) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
+| [FIPS support ](fips.md)| [7.0.4-2](https://docs.percona.com/percona-server-for-mongodb/release_notes/7.0.4-2.html) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
```
Comment (sandraromanchenko) [inline docs/psmdb-pro.md:17]:
```suggestion
| [FIPS support ](fips.md)| [8.0.1-1](https://docs.percona.com/percona-server-for-mongodb/release_notes/8.0.1-1.html) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

File: docs/install/source.md [hunk 5]
```diff
@@ -35,43 +36,56 @@ To build Percona Server for MongoDB manually, you need the following:
...
 
-=== ":material-redhat: RHEL and derivatives"
+=== ":material-debian: Debian/Ubuntu"
 
-    The following command installs the dependencies for CentOS 9:
+    The following command installs the dependencies for Ubuntu 22.04:
 
     ```{.bash data-prompt="$"}
-    $ sudo yum -y install gcc gcc-c++ cmake curl openssl-devel openldap-devel krb5-devel libcurl-devel cyrus-sasl-devel bzip2-devel zlib-devel lz4-devel xz-devel e2fsprogs-devel
+    $ gcc g++ cmake curl libssl-dev libldap2-dev libkrb5-dev libcurl4-openssl-dev libsasl2-dev liblz4-dev libpcap-dev libbz2-dev libsnappy-dev zlib1g-dev libzlcore-dev liblzma-dev ibext2fs-dev e2fslibs-dev bear
```
Comment (nastena1606) [inline docs/install/source.md:50]:
```suggestion
    $ sudo apt install -y gcc g++ cmake curl libssl-dev libldap2-dev libkrb5-dev libcurl4-openssl-dev libsasl2-dev liblz4-dev libpcap-dev libbz2-dev libsnappy-dev zlib1g-dev libzlcore-dev liblzma-dev ibext2fs-dev e2fslibs-dev bear
```

**Review by radoslawszulgo — CHANGES_REQUESTED** [review]
Please remove distro ;)

**Comment by nastena1606** [comment]
> Please remove distro ;)

@radoslawszulgo we don't list distro in this doc set

**Pull request merged**